docs/Configuration/fleet-server-configuration.md

```diff
@@ -2773,6 +2773,20 @@ The content of the Windows WSTEP identity key. An RSA private key, PEM-encoded.
       -----END RSA PRIVATE KEY-----
   ```
 
+## Partnerships
+
+### partnerships_enable_secureframe
+
+When enabled, end user's who select **Fleet Desktop > About Fleet** will be navigated to the fleetdm.com/better page with [Secureframe](https://secureframe.com/) branding. See the page here: https://fleetdm.com/better?utm_content=secureframe
```

Comment:
@noahtalerman When transparency URL is also configured on Fleet Premium, which wins?

Reply:
Shoot! Good catch. I think we have transparency URL win.

I added this^ to the test plan.

FYI @jmwatts

Reply:
Cool; that's what's currently implemented. That was the only thing holding up user story review, so moving it across to ready for spec (at which point I'll fill in the other bits).